### PR TITLE
icc-BQL add prewarning

### PR DESCRIPTION
**Pull request by Nispodzianka**
added prewarning sound+warning and InfoFrame to track Bite in raid

warnSwarmingShadowsSoon:ScheduleVoice(25.5, "specialsoon")
in 
function mod:OnCombatStart(delay)
function mod:SPELL_CAST_SUCCESS(args)
function mod:CHAT_MSG_RAID_BOSS_EMOTE(msg, _, _, _, target)

"specialsoon" needs to be changed to the correct sound

### CODE REVIEW

File: DBM-Icecrown/TheCrimsonHall/Lanathel.lua [hunk 1]
```diff
@@ -46,41 +49,55 @@ local timerEssenceoftheBloodQueen	= mod:NewBuffFadesTimer(60, 70867, nil, nil, n
...
+	warnSwarmingShadowsSoon:Schedule(25.5-delay)
+	warnSwarmingShadowsSoon:ScheduleVoice(25.5-delay, "specialsoon")
 	table.wipe(pactTargets)
 	self.vb.pactIcons = 6
 	if self.Options.RangeFrame then
 		DBM.RangeCheck:Show(8)
 	end
 	if self:IsDifficulty("normal10", "heroic10") then
 		timerNextInciteTerror:Start(124-delay)
+		warnInciteTerrorSoon:Schedule(95)
```
Comment (Zidras) [inline DBM-Icecrown/TheCrimsonHall/Lanathel.lua:87]: This is not aligned with the above timer

Comment (Nispodzianka) [reply DBM-Icecrown/TheCrimsonHall/Lanathel.lua]: fixed :)

File: DBM-Icecrown/TheCrimsonHall/Lanathel.lua [hunk 2]
```diff
@@ -19,9 +19,12 @@ mod:RegisterEvents(
 )
 
 local warnPactDarkfallen			= mod:NewTargetAnnounce(71340, 4)
+local warnPactDarkfallenSoon		= mod:NewSoonAnnounce(71340, 4)
```
Comment (Zidras) [inline DBM-Icecrown/TheCrimsonHall/Lanathel.lua:22]:
```suggestion
local warnPactDarkfallenSoon		= mod:NewSoonAnnounce(71340, 4, nil, nil, nil, nil, nil, 2)
```

File: DBM-Icecrown/TheCrimsonHall/Lanathel.lua [hunk 3]
```diff
@@ -19,9 +19,12 @@ mod:RegisterEvents(
...
 local warnBloodMirror				= mod:NewTargetAnnounce(71510, 3, nil, "Tank|Healer")
 local warnSwarmingShadows			= mod:NewTargetAnnounce(71266, 4)
+local warnSwarmingShadowsSoon		= mod:NewSoonAnnounce(71266, 4)
```
Comment (Zidras) [inline DBM-Icecrown/TheCrimsonHall/Lanathel.lua:25]:
```suggestion
local warnSwarmingShadowsSoon		= mod:NewSoonAnnounce(71266, 4, nil, nil, nil, nil, nil, 2)
```

**Review by Zidras — APPROVED** [review]

**Pull request merged**